libcaf_net/caf/net/socket_manager.hpp

```diff
@@ -49,6 +49,12 @@ class socket_manager : public ref_counted {
     return handle_;
   }
 
+  /// Returns a pointer to the multiplexer running the socket manager or
+  /// `nullptr` if the manager is no assigned to one.
```

Comment:
That sentence feels a bit unwieldy. Isn't it just trying to say something along the lines of `Returns the parent multiplexer`?

Reply:
I think so too. Maybe @josephnoir knows more than we do?

Reply:
Nope, he doesn't.